quixstreams/app.py

```diff
@@ -231,7 +232,7 @@ def _set_quix_config_builder(self, config_builder: QuixKafkaConfigsBuilder):
     @classmethod
     def Quix(
         cls,
-        consumer_group: str,
+        consumer_group: Optional[str] = "quixstreams-default",
```

Comment:
Using `Optional` is not correct here.
It means that the `consumer_group` may be `None` but the `Application` always expects a `str` type.